docs/src/drivers/gm.adoc

```diff
@@ -702,25 +676,24 @@ where <module ID> is from 00 to 15.
 .Relay output module pins
 [width="80%",options="header",cols="<3,^2,<6"]
 |===
-| Pins                | Type and direction | Pin description
-| .relay-<0-7>        | (bit, Out)         | Output pin for relay
+| Pins                 | Type and direction | Pin description
+m| .relay-<0-7>        | (bit, Out)         | Output pin for relay
 |===
 
 .Relay output module parameters
 [width="80%",options="header",cols="<3,^2,<6"]
 |===
-| Parameters          | Type and direction | Parameter description
-| .invert-relay-<0-7> | (bit, R/W)         | Negate relay output pin
+| Parameters           | Type and direction | Parameter description
+m| .invert-relay-<0-7> | (bit, R/W)         | Negate relay output pin
 |===
 
 .HAL example
-
-----
-gm.0.rs485.0.relay-0 # First relay of the node.
-gm.0                 # Means the first GM6-PCI motion control card (PCI card address = 0)
-.rs485.0             # Select node with address 0 on the RS485 bus
-.relay-0             # Select the first relay
-----
+|===
+m|gm.0.rs485.0.relay-0 | First relay of the node.
+m|gm.0                 | Means the first GM6-PCI motion control card (PCI card address = 0)
+m|.rs485.0             | Select node with address 0 on the RS485 bus
+m|.relay-0             | Select the first relay
+|===
```

Comment:
```suggestion
.First relay on RS485 node 0 on card 0
[source,{hal}]
----
gm.0.rs485.0.relay-0 # First relay of the node.
                     # gm.0 - Means the first GM6-PCI motion control card (PCI card address = 0)
                     # .rs485.0 - Select node with address 0 on the RS485 bus
                     # .relay-0 - Select the first relay
----
```

Reply:
```
.First relay on RS485 node 0 on card 0
====
[source,{hal}]
----
gm.0.rs485.0.relay-0 
----
`gm.0` Means the first GM6-PCI motion control card (PCI card address = 0)
`.rs485.0` Select node with address 0 on the RS485 bus
`.relay-0` Select the first relay
====
```
I agree, maybe the explanation doesn't have to be in the code block.

But if we have it inside the code block, we can have it like this:
```
.First relay on RS485 node 0 on card 0
====
[source,{hal}]
----
  gm.0.rs485.0.relay-0  # First relay of the node.
# gm.0                    Means the first GM6-PCI motion control card (PCI card address = 0)
#     .rs485.0            Select node with address 0 on the RS485 bus
#             .relay-0    Select the first relay
----
```

Reply:
> ```
> .First relay on RS485 node 0 on card 0
> ====
> [source,{hal}]
> ----
> gm.0.rs485.0.relay-0 
> ----
> `gm.0` Means the first GM6-PCI motion control card (PCI card address = 0)
> `.rs485.0` Select node with address 0 on the RS485 bus
> `.relay-0` Select the first relay
> ====
> ```
> 
> I agree, maybe the explanation doesn't have to be in the code block.

This may well be my preferred format :+1: , just using a `#<1><2><3>`  callouts or a simple unordered list for the 3 elements.

> But if we have it inside the code block, we can have it like this:
> 
> ```
> .First relay on RS485 node 0 on card 0
> ====
> [source,{hal}]
> ----
>   gm.0.rs485.0.relay-0  # First relay of the node.
> # gm.0                    Means the first GM6-PCI motion control card (PCI card address = 0)
> #     .rs485.0            Select node with address 0 on the RS485 bus
> #             .relay-0    Select the first relay
> ----
> ```

That's a clever one but my eyes on first sight so...

Reply:
Thought about it again last night and I'm afraid that the example block makes it standout even more than the sibling without it.
Kind of a pity because it is semantically sound. Could we make something with a pinch of CSS Fu ? Like removing bold face for example block titles ? And left side border ?